### ISSUE TITLE
Since this morning ruby dependancy crash

### ISSUE THREAD

**Issue by cokacola1978** [puphpet/puphpet#2440]
Since this morning, I can't install my VM because of the following error :  
```
==> srv70-02: ERROR:  Could not find a valid gem 'deep_merge' (= 1.0.1), here is why:
==> srv70-02:           Unable to download data from https://rubygems.org/ - SSL_connect returned=1 errno=0 state=error: certificate verify failed (https://api.rubygems.org/specs.4.8.gz)
==> srv70-02: ERROR:  Could not find a valid gem 'activesupport' (= 4.2.6), here is why:
==> srv70-02:           Unable to download data from https://rubygems.org/ - SSL_connect returned=1 errno=0 state=error: certificate verify failed (https://api.rubygems.org/specs.4.8.gz)
==> srv70-02: ERROR:  Could not find a valid gem 'vine' (= 0.2), here is why:
==> srv70-02:           Unable to download data from https://rubygems.org/ - SSL_connect returned=1 errno=0 state=error: certificate verify failed (https://api.rubygems.org/specs.4.8.gz)
```

The last night, I have could to do a good install

**Comment by ollie-new** [comment]
Just for the record - I have the exact same issue.

**Comment by adamnicolson** [comment]
me too

**Comment by jackwilsontotemic** [comment]
Yeah us here on CentOs

**Comment by MatthewBooth** [comment]
Also

**Comment by sfarkas1988** [comment]
i've the same issue, i saw an older issue where they said its not puphpet related, but a problem with the DNS or some other network thing. But for me yesterday everything went well too.
I hope this will work again, had too many issues in the last days with puphet. But at least, i really really like this tool!

**Comment by servicefinder** [comment]
Had the same problems in windows/cygwin.
Putting a '#' in front of these lines in ./puphpet/shell/install-puppet.sh "solved it" for me:
/opt/puppetlabs/puppet/bin/gem` install deep_merge -v 1.0.1 --no-ri --no-rdoc
/opt/puppetlabs/puppet/bin/gem install activesupport -v 4.2.6 --no-ri --no-rdoc
/opt/puppetlabs/puppet/bin/gem install vine -v 0.2 --no-ri --no-rdoc

Manually updating the gems first might be a good idea:
`gem install deep_merge`
`gem install activesupport`
`gem install vine`

**Comment by daneelgiskard** [comment]
Just another "me too", all of my colleagues have this problem too. I guess something changed at rubygems.org regarding the SSL certificates.

**Comment by jtreminio** [comment]
They are aware of the issue: https://github.com/rubygems/rubygems/issues/1736

The commit above will completely resolve the problem, though.

**Comment by jackwilsontotemic** [comment]
thanks

**Comment by therobyouknow** [comment]
> @jtreminio jtreminio added a commit that closed this issue 
>  @jtreminio	Only install required gems if missing; fixes #2440

I found the above change in the commit to work, the link given is also included below

https://github.com/puphpet/puphpet/commit/a905085f8f4ac3fc00d51d2a348fc8b02e52f67d

Background: only for info if interested: I needed to do this when modifying a puphpet.zip from 2016 09 09 that had MySQL 5.6 - I needed it to use MySQL 5.5 (for the purposes of investigating a theory I had about a potential issue with Drupal rabbit_hole module on different MySQL versions )

Thank you @jtreminio
